PeerioTechnologies/peerio-desktop
# feat: retry loading images after connection restored

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### PR author

#### Relevant issue tracker links  
https://peerio.tpondemand.com/RestUI/Board.aspx#page=bug/7463&appConfig=eyJhY2lkIjoiMTA1MTA5MDU0OEY2QTUyQjlFM0JCODkwRjYwQUVGMEIifQ==
#### Linked PRs in other repos
no 
#### Manual testing instructions  
1. open app, close internet connection before app fully loaded
2. images should be replaced by 'broken link' symbol
3. open internet connection, images should reload automatically
#### Additional notes
Applied the changes in other places images are loaded from urls, not just avatars.

----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
